client/src/views/CellView.tsx

```diff
@@ -66,6 +66,27 @@ const CellView = () => {
     });
   }, [Alert, Confirm, brute, navigate, t, updateData]);
 
+  const switchBrute = useCallback((side : number) => {
+    if (!user || !brute || !user.brutes) return;
```

Comment:
Add a check with ownsBrute here, as we don't need to do another findIndex if we don't own the brute